pkg/varnishcluster/controller/varnishcluster_statefulset_test.go

```diff
@@ -101,7 +101,7 @@ var _ = Describe("statefulset", func() {
 			Expect(varnishContainer.Args).To(Equal([]string{
 				"-F",
 				"-S", "/etc/varnish-secret/secret",
-				"-T", "127.0.0.1:6082",
+				"-T", "0.0.0.0:6082",
```

Comment:
Can you explain in more detail how changing the address from `127.0.0.1` to `0.0.0.0` exposes the management port in a secure fashion? I understand that `127.0.0.1` (localhost) is only accessible from inside the pod, but doesn't this change allow anyone to hit port 6082?

Reply:
Yes, it allows anyone inside the same network to access that management port, but it is still protected by the authentication by secret (the `"-S", "/etc/varnish-secret/secret"` parameter). The client should know it before establishing the connection.